app/assets/javascripts/geoblacklight/viewers/esri.js

```diff
@@ -21,7 +21,7 @@ GeoBlacklight.Viewer.Esri = GeoBlacklight.Viewer.Map.extend({
     // remove any trailing slash from endpoint url
     _this.data.url = _this.data.url.replace(/\/$/, '');
 
-    L.esri.get = L.esri.Request.get.JSONP;
+    // L.esri.get = L.esri.Request.get.JSONP;
```

Comment:
I think it's OK to remove this line entirely. Documentation on the ESRI Leaflet tool says that fallback to JSONP is handled automatically now.